testing/compile.ts

```diff
@@ -103,10 +110,23 @@ async function uglifyBundle({
   fs.writeFileSync(`${outputBundleFilename}.map`, uglifyOutput.map);
 }
 
+function getInjections({timerStrategy = TimerShimStrategy.Disable}: CompilePackBundleOptions) {
+  switch (timerStrategy) {
+    case TimerShimStrategy.Disable:
+      return [];
```

Comment:
When I saw the option "Disable", I thought that mean, disable timers, but I think it means, don't use a shim? In which case I think `TimerShimStrategy.None` is maybe clearer?